llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -401,6 +408,14 @@ def FeatureStdExtZcf
                      "Compressed Single-Precision Floating-Point Instructions",
                      [FeatureStdExtF, FeatureStdExtZca]>;
 
+def FeatureStdExtZclsd
+    : RISCVExtension<1, 0,
+                     "Compressed Load/Store pair instructions",
+                     [FeatureStdExtZilsd,FeatureStdExtZca]>;     
+def HasStdExtZclsd : Predicate<"Subtarget->hasStdExtZclsd() && !Subtarget->hasStdExtZcf()">,
```

Comment:
The conflict with Zcf needs to be checked in `RISCVISAInfo::checkDependency` in llvm/lib/TargetParser/RISCVISAInfo.cpp